src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -634,6 +658,9 @@ public override void Accept(OperationVisitor visitor)
 
     internal partial class BoundLabelStatement : ILabelStatement
     {
+        // These represent synthesized labels, and do not have an attached statement.
+        IOperation ILabelStatement.LabeledStatement => null;
```

Comment:
No. This was as close as I could get to unifying the treatment of labels in C# and VB. This property is mostly to make the walker work.

For the C# fragment:
    Target: if (...);
    return 3;
the C# compiler generates 2 statements in the containing block. The first is a labeled statement that has an if statement as an operand. The second is a return statement.

For the equivalent VB fragment, the VB compiler generates 3 statements in the containing block. The first is a labeled statement with no operand, the second is an if statement, and the third is a return statement.

An analyzer should generally ignore the statement operand of a label statement, but the walker needs access to it in order to expose the labeled statement to analyzers.